test/runnable/traits.d

```diff
@@ -1559,7 +1597,8 @@ int main()
     test4();
     test5();
     test6();
-    test7();
+    test7a();
+    test7b();
```

Comment:
Is this ok? Maybe I'm missing something as I don't know much about the test system in dmd, but won't this recursively call `test7b`, generating a `stack limit exceeded` error?

Reply:
Why would it call it recursively? Just expand the code a bit... this is in main().

Reply:
Whops :laughing: